app/views/like/_like.html.erb

```diff
@@ -16,10 +16,10 @@
 
   <li rel="tooltip" title="Helpful? Like it and get updates!" class="btn btn-default btn-sm btn-like" node-id="<%= node.id %>" id="like-button-<%= node.id %>">
     <% if !current_user %>
-      <%= link_to  new_user_session_path( return_to: request.path ) do %>
+      <a id="open-login-like" data-hashparams="like" data-toggle="modal" data-target="#loginModal">
```

Comment:
Hey! @okonek I'm working on another issue that uses this, I'm just wondering, what does the hashparams do?

Reply:
@dinaelhanan you will not be needing them I guess in that pr. You need the changes similar to the other file changes.

Reply:
@SidharthBansal I thought that I would be changing the link here:
https://github.com/publiclab/plots2/blob/5c0d96274bd0be15feba69d8f30093082402ace5/app/views/dashboard/_header.html.erb#L39
into the link that okonek added above

Reply:
Yeah. Please make the conversation in the issue for that issue. Don't link unrelated prs and issues.
I hope you can understand it.